src/main/java/uk/ac/cam/cl/quebec/face/FaceDaemon.java

```diff
@@ -43,34 +43,43 @@ private boolean setQueuePass(String password)
 
     private void connectToQueue()
     {
-        tempQueue.add(new AddPhotoMessage(0, 3, "img/training/0/0.jpg"));
-        tempQueue.add(new AddPhotoMessage(1, 3, "img/training/0/1.jpg"));
+        tempQueue.add(new AddPhotoMessage(0, 0, "img/training/0/0.jpg"));
+        tempQueue.add(new AddPhotoMessage(1, 0, "img/training/0/1.jpg"));
+        tempQueue.add(new AddPhotoMessage(2, 0, "img/training/0/2.jpg"));
+        tempQueue.add(new AddPhotoMessage(3, 0, "img/training/0/3.jpg"));
+        tempQueue.add(new AddPhotoMessage(4, 0, "img/training/0/4.jpg"));
+        tempQueue.add(new AddPhotoMessage(10, 1, "img/training/1/0.jpg"));
+        tempQueue.add(new AddPhotoMessage(11, 1, "img/training/1/1.jpg"));
+        tempQueue.add(new AddPhotoMessage(12, 1, "img/training/1/2.jpg"));
         Set<Integer> photos1 = new HashSet<>();
         photos1.add(0);
-        tempQueue.add(new ProcessVideoMessage(11, "img/video/1.mp4", photos1));
+        tempQueue.add(new ProcessVideoMessage(11, "img/video/0.mp4", photos1));
     }
 
-    private Message getJobFromQueue()
+    private Message getJobFromQueue() throws FaceException
```

Comment:
The intention is for it to block until the queue is not empty - the exception is there for if something goes properly wrong (queue connection dies, etc)